inttests/sds_test.go

```diff
@@ -120,3 +120,66 @@ func TestCreateSdsInvalid(t *testing.T) {
 	assert.Equal(t, "", sdsID)
 
```

Comment:
can you also please cover the valid scenarios. I saw you only covered invalid scenarios. Thank you

Reply:
I intentionally kept it like this because, valid case will cause changes in actual data as it is integration test, I have tested it manually but not put it into code, as it will hamper existing data integrity.

Reply:
okay